fuel/app/tasks/robots.php

```diff
@@ -2,58 +2,56 @@
 /**
  * Fuel is a fast, lightweight, community driven PHP5 framework.
  *
- * @package    Fuel
  * @version    1.7
+ *
  * @author     Fuel Development Team
  * @license    MIT License
  * @copyright  2010 - 2015 Fuel Development Team
+ *
  * @link       http://fuelphp.com
  */
-
 namespace Fuel\Tasks;
 
 /**
- * Robot example task
+ * Robot example task.
  *
  * Ruthlessly stolen from the beareded Canadian sexy symbol:
  *
  *		Derek Allard: http://derekallard.com/
  *
- * @package		Fuel
  * @version		1.0
+ *
  * @author		Phil Sturgeon
  */
-
-class Robots
+class robots
 {
-	/**
-	 * This method gets ran when a valid method name is not used in the command.
-	 *
-	 * Usage (from command line):
-	 *
-	 * php oil r robots
-	 *
-	 * or
-	 *
-	 * php oil r robots "Kill all Mice"
-	 *
-	 * @return string
-	 */
-	public static function run($speech = null)
-	{
-		if ( ! isset($speech))
-		{
-			$speech = 'KILL ALL HUMANS!';
-		}
+    /**
+     * This method gets ran when a valid method name is not used in the command.
+     *
+     * Usage (from command line):
+     *
+     * php oil r robots
+     *
+     * or
+     *
+     * php oil r robots "Kill all Mice"
+     *
+     * @return string
+     */
+    public static function run($speech = null)
+    {
+        if (!isset($speech)) {
+            $speech = 'KILL ALL HUMANS!';
+        }
 
-		$eye = \Cli::color("*", 'red');
+        $eye = \Cli::color('*', 'red');
 
-		return \Cli::color("
+        return \Cli::color("
 					\"{$speech}\"
 			          _____     /
 			         /_____\\", 'blue')."\n"
-.\Cli::color("			    ____[\\", 'blue').$eye.\Cli::color('---', 'blue').$eye.\Cli::color('/]____', 'blue')."\n"
-.\Cli::color("			   /\\ #\\ \\_____/ /# /\\
+.\Cli::color('			    ____[\\', 'blue').$eye.\Cli::color('---', 'blue').$eye.\Cli::color('/]____', 'blue')."\n"
```

Comment:
__[phpmd]__
* Avoid using static access to class '\Cli' in method 'run'.
* Avoid using static access to class '\Cli' in method 'run'.
* Avoid using static access to class '\Cli' in method 'run'.